public/admin/config.yml

```
@@ -62,3 +62,30 @@ collections:
               required: false,
               hint: "Cloudinary image path, eg. `v12345/my_file.png`",
             }
+  - name: "isoman_page"
+    label: "Isoman Page"
+    delete: false
+    preview_path: ""
```

```suggestion
    preview_path: "isolasi-mandiri"
```

This field is meant for the editors so that they can go to the correct page when they click the `Preview page` link on Netlify CMS.